apps/server/src/schema/feedback/feedback-operation.ts

```diff
@@ -0,0 +1,55 @@
+import { FeedbackType, prisma } from '@repo/database';
+import { FireAndForget } from '@repo/utils';
+import { GraphQLError } from 'graphql';
+import { builder } from '../builder';
+import { sendFeedbackReceivedEmail } from '../../email';
+import { FeedbackDto, FeedbackTypeEnum } from './feedback-types';
+
+const fireAndForget = new FireAndForget();
+
+builder.mutationFields((t) => ({
+  sendFeedback: t.withAuth({ authenticated: true }).field({
+    nullable: false,
+    type: FeedbackDto,
+    args: {
+      message: t.arg.string({ required: true }),
+      type: t.arg({
+        type: FeedbackTypeEnum,
+        required: true,
+      }),
+    },
+    validate: (args) => Boolean(args.message && args.type),
+    resolve: async (_root, args, ctx, _info) => {
```

Comment:
```suggestion
    resolve: async (query, _root, args, ctx, _info) => {
```
since we are returning a prismaField, pothos add query as the first parameter of resolve

Reply:
`add query as the first parameter of resolve` @aaryansinha16